lib/dependencies/SystemPlugin.js

```diff
@@ -39,6 +39,11 @@ class SystemPlugin {
 						parser.state.module.context, require.resolve("../../buildin/system.js"));
 					return ParserHelpers.addParsedVariableToModule(parser, "System", systemPolyfillRequire);
 				});
+
+				parser.hooks.call.for("System.import").tap("SystemPlugin", expr => {
+					parser.hooks.importCall.call(expr);
```

Comment:
This trick removes `System.import` handling from `ImportPlugin` by simulating an `import()` call.